src/Raven.Server/Documents/Handlers/Admin/AdminRevisionsHandler.cs

```diff
@@ -18,12 +18,14 @@ public class AdminRevisionsHandler : DatabaseRequestHandler
         [RavenAction("/databases/*/admin/revisions", "DELETE", AuthorizationStatus.DatabaseAdmin)]
         public async Task DeleteRevisionsFor()
         {
+            var deleteOnlyForceCreated = GetBoolValueQueryString("onlyForceCreated", required: false) ?? false;
```

Comment:
No, if it's true it deletes only force-created revisions.
I did it to let the users have the option to remove only force-created revisions.
The only other way to remove them is by enforce-revisions-configuration (including force-created revisions), and it won't necessarily delete them either.
You asked me for something like that..

Reply:
okay, I think it would be better to allow users to delete a specific revision by proving a change-vector(s)
Do we have something like that? if not, let's create a separate issue to handle that, since this affect the client as well.

Reply:
Ok

Reply:
It now has url parameter called "includeForceCreated" which is telliting if delete all revisions including or excluding the force created revisions